scripts/restore

```diff
@@ -31,8 +31,10 @@ db_name=$(ynh_app_setting_get $app db_name)
 # CHECK IF THE APP CAN BE RESTORED
 #=================================================
 
-CHECK_DOMAINPATH	# Check domain and path availability
-CHECK_FINALPATH # Check if destination directory is not already in use
+yunohost app checkurl "${domain}${path_url}" -a "$app" \
```

Comment:
Here we could use [ynh_webpath_available](https://github.com/YunoHost/example_ynh/blob/master/scripts/restore#L39).